bitnami/postgresql-repmgr/12/debian-11/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -237,6 +249,56 @@ repmgr_get_primary_node() {
     echo "$primary_port"
 }
 
+
+
+########################
+# switchover standby after long primary wait
+# Globals:
+#   REPMGR_*
+#   POSTGRESQL_*
+# Arguments:
+#   None
+# Returns:
+#   None
+#########################
+
+repmgr_switchover_standby() {
+    local -i timeout="${REPMGR_WAIT_PRIMARY_TIME:-60}"
+
+    primary_host=""
+    primary_port="${REPMGR_PORT_NUMBER}"
+
+info "Primary node is unavailable. Assuming primary role..."
+
+    touch "$REPMGR_PROMOTE_STANDBY_LOCK_FILE_NAME"
+
+    debug "Marking node to be promoted from standby to primary by writing the following file: '$REPMGR_PROMOTE_STANDBY_LOCK_FILE_NAME' "
+
+    [[ -n "$primary_host" ]] && debug "Primary node: '${primary_host}:${primary_port}'"
```

Comment:
Please keep an eye on the coding style. Be careful with indentation and additional white lines.
```suggestion
    primary_host=""
    primary_port="${REPMGR_PORT_NUMBER}"
    
    info "Primary node is unavailable. Assuming primary role..."
    touch "$REPMGR_PROMOTE_STANDBY_LOCK_FILE_NAME"
    debug "Marking node to be promoted from standby to primary by writing the following file: '$REPMGR_PROMOTE_STANDBY_LOCK_FILE_NAME' "
    [[ -n "$primary_host" ]] && debug "Primary node: '${primary_host}:${primary_port}'"
```